raiden/network/proxies/token_network.py

```diff
@@ -1425,7 +1425,8 @@ def close(
         balance_hash: BalanceHash,
         nonce: Nonce,
         additional_hash: AdditionalHash,
-        signature: Signature,
+        non_closing_signature: Signature,
```

Comment:
As discussed in the chat, we have a code freeze for the release. I would like to avoid any unnecessary changes for the release at the moment. So unless this change is needed for this release I would like to avoid it.

**important note**: I can't find anything wrong with this code in my review though.

But the proxies are used everywhere so it's not a small client change and bug can sip in at any point.

Reply:
@LefterisJP it's possible to remove this change.  `TokenNetwork` just needs to leave `closeChannel()` unchanged, and add a new `closeChannelByThirdParty()`.

Reply:
@LefterisJP shall I do ^?

Reply:
@Dominik1999 said I should keep this.